docker-compose.yaml

```diff
@@ -0,0 +1,38 @@
+version: '3.9'
```

Comment:
Great! now we can test in docker way more easy locally, thanks!
Is there a way to make this available for every app ? otherwise I'll move to the app folder and duplicate in fishing-map when needed

Reply:
I added `map` to the docker-compose, now it can be used too 😉 . The idea is to use this single docker-compose and add inside it all our apps in the monorepo to be proxied like we do in prod.